# twins_snp

**fastqc** (ver. 0.12.1; [www.bioinformatics.babraham.ac.uk/projects/fastqc/](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/))   
**fastp** (ver. 0.23.2; https://doi.org/10.1002/imt2.107)   
**samtools** (ver. 1.13; https://doi.org/10.1093/bioinformatics/btp352)   
**Genome Analysis Tool Kit (GATK)** (ver. 4.6.0; https://doi.org/10.1101/201178)   
**DeepVariant** (ver 1.8; https://doi.org/10.1038/nbt.4235)   
**GLnexus** (ver 1.4.1; https://doi.org/10.1093/bioinformatics/btaa1081, https://doi.org/10.1101/343970)   
**bcftools** (ver 1.11; https://doi.org/10.1093/gigascience/giab008)   
   
   
   
참고1 : **fastq 검증 ~ variant calling 까지의 과정의 예시**   
https://www.science.org/doi/10.1126/sciadv.aaz7835#sec-4   
KOREA1K 논문입니다.   
   
참고2: **Mills_and_1000G_gold_standard.indels.hg38.vcf.gz** 와 **Homo_sapiens_assembly38.dbsnp138.vcf.gz**   
은 reference 를 달지 않아도 될 것 같습니다. 해당 파일들의 citation 은 보통 잘 하지 않는 편인 것 같습니다.   
   
참고3: DeepVariant 가 공식 명칭으로 보입니다.   (https://github.com/google/deepvariant?tab=readme-ov-file)

합쳐진 큰그림     (원본: [Up)loading twin_overall_bioinformatic_workflow1.drawio…]()

![image](https://github.com/user-attachments/assets/2ac61035-e421-408d-bdc9-1769ca72c6c8)

![image](https://github.com/user-attachments/assets/3b9351a4-7e86-4836-9a93-f1b8d1d4c471)

